test/unit/whitehall/asset_manager_storage_test.rb

```diff
@@ -53,7 +54,7 @@ class AssetManagerUploader < CarrierWave::Uploader::Base
     storage.retrieve!('identifier')
   end
 
-  test 'returns an asset manager file' do
+  test 'retrieve! returns an asset manager file' do
```

Comment:
Minor: Ideally this would be in a separate commit.

Reply:
Done.